plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-logo.tsx

```diff
@@ -451,7 +451,8 @@ export const SidebarNavigationScreenLogo = () => {
 								popoverProps={ {
 									className:
 										'woocommerce-customize-store__logo-dropdown-popover',
-									position: 'bottom left',
+									// @ts-expect-error outdated TS.
+									placement: 'left',
```

Comment:
should it be `left-end`?

Reply:
Actually not sure how I ended up with `left`, this should be `bottom-end`

Reply:
Updated here: https://github.com/woocommerce/woocommerce/pull/41845/commits/9dce5272310c86c2477551315fd75c516e26b22c